sparkmagic/example_config.json

```diff
@@ -65,5 +65,6 @@
   "livy_server_heartbeat_timeout_seconds": 0,
   "heartbeat_retry_seconds": 10,
 
-  "server_extension_default_kernel_name": "pysparkkernel"
+  "server_extension_default_kernel_name": "pysparkkernel",
+  "authentication_cookie_name": "hadoop.auth"
```

Comment:
Can you please add instructions (and a link to the proper livy conf) to the README on how to use this new setting you are introducing?

If this is a new form of authentication, maybe you should add an ldap authentication option besides the existing ones. What do you think?